README.md

```diff
@@ -55,14 +55,65 @@ The source code in this package is released under the Apache-2.0 License. For fu
 The [3rdparty-licenses.txt](3rd-party-licenses.txt) contains a list of other open source components included in this package.
 
 
-## Installation & Usage as a ROS package
+## Installation & Usage as a [ROS2] package
 
 ### Dependencies
```

Comment:
This section is repeated 3 times in the README. I suggest that we have a separate section ##Dependencies before the ##Installation & Usage section

Reply:
It's not fully repeated. For ROS 1 & 2 you can use rosdep to install the dependencies. If you don't use ros you need to apt-get install.

Reply:
Can we still introduce a section ##Dependencies before ##Installation with the apt command for libeigen? We can then have ### ROS 2 Dependencies in the ##Installation section.

Reply:
I've moved this section to the top and made a split for ROS vs non ROS